### PR TITLE
Fix for endpoint api selection.

### PR DESCRIPTION
**Pull request by kfletche**
Quick fix to allow api selection outside the european endpoint.

### CODE REVIEW

**Comment by yadutaf** [comment]
Oh! you're completely right. I'm sorry I forgot this one... Would you mind adding a CLI flag along with its documentation ?

**Comment by kfletche** [comment]
Give it a try now.

**Comment by yadutaf** [comment]
It works perfectly. Thanks! You could even set the default to empty string like for ``--ovh-consumer-key``, this should trigger the default path in https://github.com/ovh/go-ovh/blob/master/ovh/configuration.go#L66 and bring support for the environment (``$OVH_CONSUMER_KEY``) and conf file (``.ovh.conf``) based API configuration ;)

I can do this if you want.

File: driver.go [hunk 1]
```diff
@@ -73,6 +74,11 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 			Value:  "",
 		},
 		mcnflag.StringFlag{
+			Name:  "ovh-endpoint",
+			Usage: "OVH Cloud API endpoint. Default: ovh-eu",
+			Value: DefaultEndpoint,
```
Comment (yadutaf) [inline driver.go]: You can set it to ``""``` to automatically get all the validation and defaults from the API wrapper.

**Comment by kfletche** [comment]
Good call. You clearly know your api wrapper better than I :)

**Comment by yadutaf** [comment]
Thanks a lot ! I'll add the tiny missing bit in the doc later today.

**Pull request merged**